### ISSUE TITLE
style prop applied to SvgIcon generating warning

### ISSUE THREAD

**Issue by danieldai** [stowball/react-native-svg-icon#13]
I want to adjust the position of icon with `style`
```
<Icons style={styles.place} name="Place" width="12" height="12"/>
```
```
const styles = StyleSheet.create({
  place: {
    marginTop: 4,
    marginLeft: 2,
  },
})
```

the position is changed, but it generates some annoying warning:
```
Warning: Failed prop type: Invalid prop `style` supplied to `SvgIcon`.
```

**Comment by danieldai** [comment]
the work around is use in this way

```
<Icons style={[styles.place]} name="Place" width="12" height="12"/>
```

or

```
<Icons style={{marginTop: 4, marginLeft: 2,}} name="Place" width="12" height="12"/>
```

**Comment by stowball** [comment]
Sorry, I apparently missed this for 6 months :-\. Will fix!

**Comment by stowball** [comment]
I just published 0.8.1 which fixes this issue
